lib/cictl/logger.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "yell"
```

Comment:
`yell` is unsupported these days. Probably better off using plain old `logger` from teh stdlib, or maybe `semantic_logger`

Reply:
Bill will address the yell-to-semantic switchover.

Reply:
Nope, I will. And did: PR #31